### ISSUE TITLE
Recieved 'socket hang up' for 'finance/transactions/any'

### ISSUE THREAD

**Issue by Qrysto** [Nexusoft/LLL-TAO#108]
I'm now getting 'socket hang up' error for all 'finance/transactions/any' calls. I think it still worked not long ago

**Comment by Qrysto** [comment]
It occurs for all my account including trust (with `address` input), also for all `verbose` levels

**Comment by Qrysto** [comment]
There's an error in the log when the call is made

```
[19:28:31.743] Socket : SSL Connection using TLS_AES_128_GCM_SHA256
[19:28:31.743] ListeningThread : Accepted Connection 127.0.0.1:57013 on port 7080
[19:28:31.744] ERROR: Thread : Data Connection: [json.exception.type_error.302] type must be string, but is object
[19:28:31.870] Attempt : Connection failed 149.167.135.89:9888
[19:28:31.879] Manager : Tritium C=9 D=2 F=140 | TC=175478 TD=158117 TF=2608528 | B=0 | EID=0 | size=20321
[19:28:32.898] Manager : Tritium Attempting Connection 77.254.187.20:9888
```

**Comment by VidereLicet** [comment]
Resolved with commit 2fe6d7be0185c01418fa45ca4a820f7a6ba89717
